src/ZODB/Connection.py

```diff
@@ -1020,9 +1018,10 @@ def _cache_items(self):
         items = self._cache.lru_items()
         # fine everything. some on the lru list, some not
         everything = self._cache.cache_data
-        # remove those items that are on the lru list
+        # remove those items that are on the lru list (which may not actually
+        # be in the full cache, under the Python implementation)
```

Comment:
Is this change stlil true (i.e., can the Python implementation of picklecache return items from `lru_items` which are not in the cache?)